Fix flickering in Blinkenlights UI

iliastsi opened the pull request:
### This PR

> Add character x `[x]`.

- [x] I've read the [DCO](http://www.offlineimap.org/doc/dco.html).
- [x] I've read the [Coding Guidelines](http://www.offlineimap.org/doc/CodingGuidelines.html)
- [x] The relevant informations about the changes stands in the commit message, not here in the message of the pull request.
- [x] Code changes follow the style of the files they change.
- [x] Code is tested. It works for me and @chneukirchen. Since there were no more replies, I am closing both #160 and #290.

### References

- Issue #160
- Issue #290

### Additional information

Do not call resizeterm(), unless is_term_resized() returns True. This
breaks the busy-loop where resizeterm() pushes a KEY_RESIZE onto the
FIFO causing the screen to be redrawn indefinitely (Issue #290).

Also, clear and refresh the main window after it has been resized. This
hopefully fixes the problem where Blinkenlights UI becomes unreadable
after terminal resize (Issue #160).

Closes #160: blinkenlights display is broken
Closes #290: ncurses flicker with blinkenlights UI
Bug-Debian: https://bugs.debian.org/671087
Bug-Debian: https://bugs.debian.org/809676
Signed-off-by: Ilias Tsitsimpis <i.tsitsimpis@gmail.com>

nicolas33 commented:
Applied. Good job @iliastsi !

nicolas33 commented:
> Since there were no more replies, I am closing both #160 and #290.

I guess you could not close those because of privileges issues. Hence, I've added you to the official contributors to get write access to the repositories in order to manage issues. Please, tell me or send me a mail if you'd rather to decline the granting.

iliastsi commented:
> I guess you could not close those because of privileges issues. Hence, I've added you to the official contributors to get write access to the repositories in order to manage issues. Please, tell me or send me a mail if you'd rather to decline the granting.

I didn't try to close them directly, I just added the 'Closes' word on the commit message (and let Github do the [rest](https://help.github.com/articles/closing-issues-via-commit-messages/)). Thanks for adding me to the official contributors. I will continue to send my patches through pull requests, since I like the review process (if that's ok with you).

nicolas33 commented:
> I will continue to send my patches through pull requests, since I like the review process (if that's ok with you).

Didn't work. Perhaps because I'm not merging into master.

> I didn't try to close them directly, I just added the 'Closes' word on the commit message (and let Github do the rest).

I agree that's better. ,-)
